phylogenetic/defaults/config.yaml

```diff
@@ -4,11 +4,13 @@ sequences_url: "https://data.nextstrain.org/files/workflows/mumps/sequences.fast
 metadata_url: "https://data.nextstrain.org/files/workflows/mumps/metadata.tsv.zst"
 
 strain_id_field: "accession"
-reference: "defaults/reference.gb"
+reference: "reference.gb"
 
 filter:
   min_length: 8000
   group_by: country year month MuV_genotype division
+  exclude: "{build}/exclude.txt"
```

Comment:
It just occurred to me that the user has no way to know what `{build}` should be unless they look at the defined builds in the Snakefile: https://github.com/nextstrain/mumps/blob/41654ec8e601f4669146d4c3ac1834ae7f74c12f/phylogenetic/Snakefile#L11

Seems like `builds` should be moved to config params and then document here that the `{build}` in the file path needs to match the defined builds.

Reply:
Agree, I can't believe I missed this. Thanks and fixed in https://github.com/nextstrain/mumps/pull/36/commits/c9fa9ca6f773eb90a4093460137e40907a7e62dc